src/components/MyListExportActions/index.js

```diff
@@ -2,22 +2,35 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import Button from '../Button'
 import classnames from 'classnames'
+import { select, t } from '@lingui/macro'
 
 const MyListExportActions = ({ confirmDeleteAll, downloadCsv, emailList, isDownloading }) => (
   <div className='mylist__export-actions'>
     <Button
       className='btn--orange btn--sm'
-      label='Email List'
+      label={t({
+        comment: 'Email list label',
+        message: 'Email List'
+      })}
       iconBefore='email'
       handleClick={emailList} />
     <Button
       className={classnames('btn--orange', 'btn--sm', {'loading-dots': isDownloading})}
-      label={isDownloading ? 'Downloading' : 'Download as .CSV'}
+      label={t({
+        comment: 'Downloading label',
+        message: select(isDownloading, {
+          true: 'Downloading',
+          other: 'Download as .CSV'
+        })
+      })}
```

Comment:
As noted above, this feels a lot cleaner, and keeps all the translation stuff together neatly.